migrations/m3_acp_module.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ *
+ * Ideas extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2015 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\ideas\migrations;
+
+class m3_acp_module extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array('\phpbb\ideas\migrations\m2_initial_data');
+	}
+
+	public function update_data()
```

Comment:
Should have an effectively installed check... Can just check if one of the config's already exist.